pkg/controller/image-stream-tag-reconciler/reconciler.go

```diff
@@ -278,11 +315,12 @@ func (r *reconciler) createBuildForIST(job *prowconfig.Postsubmit, headSHA strin
 		Repo:    istRef.repo,
 		BaseRef: istRef.branch,
 		BaseSHA: headSHA,
-	}), nil, nil)
+	}), r.createdProwJobLabels, nil)
 	prowJob.Namespace = r.prowJobNamespace
 
 	if r.dryRun {
-		r.log.Infof("Not creating %s prowjob because dryRun is enabled", prowJob.Spec.Job)
+		serialized, _ := json.Marshal(prowJob)
+		r.log.Infof("Not creating %s prowjob because dryRun is enabled, job: %s", prowJob.Spec.Job, serialized)
```

Comment:
If you add it as a `WithField` it will print valid JSON to the command line, whereas this will escape it, harder to copy-paste and read